Q_hub/static/style.css

```diff
@@ -0,0 +1,83 @@
+nav{
```

Comment:
Sir do you mean I should exclude the templates and static folder while making the PR? because if you do then i will have to look for how to do it